books/data-structures/list-defthms.lisp

```diff
@@ -929,18 +929,19 @@
   (implies (<= (len l) (nfix n))
            (equal (nth n l) nil)))
 
-(defthm nth-append
-  (equal (nth n (append a b))
-         (if (< (nfix n) (len a))
-             (nth n a)
-             (nth (- (nfix n) (len a)) b))))
-
-(defthm nth-revappend
-  (equal (nth n (revappend a b))
-         (if (< (nfix n) (len a))
-             (nth (- (len a) (1+ (nfix n))) a)
-           (nth (- (nfix n) (len a)) b)))
-  :hints (("Goal" :induct (revappend a b))))
+;; Renamed for consistency with std/lists/nth.lisp
+(defthm nth-of-append
+  (equal (nth n (append x y))
+         (if (< (nfix n) (len x))
+             (nth n x)
+           (nth (- n (len x)) y))))
+
+;; Renamed for consistency with books/std/lists/nth.lisp
```

Comment:
I suggest changing this to read "std/lists/nth.lisp", or changing the comment on the previous theorem to read "books/std/lists/nth.lisp". Actually I suggest putting these comments in your commit message body rather than in the file -- a future reader won't care that the names of the theorems used to be different (and your comment doesn't tell them what the names used to be anyway).